e3db/types/encyption_key_pair.py

```diff
@@ -0,0 +1,20 @@
+class EncryptionKeyPair(): 
+    
+    def __init__(self, public_encryption_key, private_encryption_key):
+        """
+        Base64URL encoded representation of a public/private keypair
+
+        Parameters
+        ----------
+        public_encyption_key : str
```

Comment:
We probably don't need to specify encryption in this field. We know it is an encryption key given the class it is in, same with signing

Reply:
Length and format validation forthcoming, too.